apple-pie/index.html

```diff
@@ -4,36 +4,53 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet"  href="/apple-pie/styles/style.css">
   </head>
   <body>
-    Apple Pie
-
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
-    
-    Ingredients 
+    <div class = "tile-background-image">
+      <h1>Apple Pie</h1>
+    </div>
     
-    1 recipe pastry for a 9 inch double crust pie 
-    1/2 cup unsalted butter 
-    3 tablespoons all-purpose flour
-    1/4 cup water
-    3 tablespoons all-purpose flour
-    1/4 cup water 1/2 cup white sugar
-    1/2 cup packed brown sugar
-    8 Granny Smith apples - peeled, cored and sliced
+    <div class = "format">
+      <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+        be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
+        favorites as well! </p>
+    </div>
 
-    Directions
+    <div>
+      <img class = "image-format" src="/french-soup/images/recipe-info.png">
```

Comment:
```suggestion
      <img class="image-format" src="images/recipe-info.png">
```
Por que buscas en la carpeta de french-soup, si estamos en apple-pie ??